.github/pull_request_template.md

```diff
@@ -1,10 +1,18 @@
-**Related Issue(s)**:  ...
+### Related Issues
+- fixes #issue-number
 
-**Proposed changes**:
-- ...
+### Proposed Changes?
+ <!--- In case of a bug: Describe what caused the issue and how you solved it-->
```

Comment:
Minor: I would add another line with instruction for features, like 
```suggestion
 <!--- In case of a bug: Describe what caused the issue and how you solved it -->
 <!--- In case of a feature: Describe what did you add and how it works -->
```